api-reference/v2/tables/get-table-rows.mdx

```diff
@@ -2,3 +2,7 @@
 title: Get Rows
 openapi: get /tables/{tableID}/rows
 ---
+
+You can use the `limit` query parameter to specify the maximum number of rows to return for each request.
+
+Whether or not you supply a `limit`, you may need to make multiple requests to retrieve all the rows in the table. If there are more rows to fetch, the `continuation` field will be set in the response. To retrieve the next page of rows, make another request with the `continuation` as a query parameter.
```

Comment:
I wold love to see us add a tutorial page (similar to https://apidocs.glideapps.com/api-reference/v2/tutorials/bulk-import) that walks through the steps on how to iterate through a large dataset using limits/continuations. That's the type of multi-step workflow that is best to see in sequence rather than just a reference of the API itself.

Reply:
Agreed!